yt/chyt/controller/internal/strawberry/oplet.go

```diff
@@ -888,13 +979,25 @@ func (oplet *Oplet) updateOpParameters(ctx context.Context, reason string) error
 	oplet.l.Info("updating operation parameters", log.String("reason", reason))
 
 	opACL := oplet.getOpACL()
+	params := map[string]any{
+		"acl":  opACL,
+		"pool": oplet.strawberrySpeclet.Pool,
+	}
+	if oplet.pendingScaling && oplet.targetInstanceCount != 0 {
+		params["scheduling_options_per_pool_tree"] = map[string]any{
+			"default": map[string]any{
+				"resource_limits": map[string]any{
+					"user_slots": oplet.targetInstanceCount,
+				},
+			},
+		}
+		oplet.pendingScaling = false
```

Comment:
To the end of the function (so only update if there is no error in `UpdateOperationParameters` call).
Also it's possible to eliminate `pending` flag at all (e.g. we do not have pending flag for `acl`), but it's not required, you can leave it as is